docs/install.md

```diff
@@ -150,6 +138,122 @@ on the Kubernetes version.
 $ kubectl label node <your node> storage=pmem
 ```
 
+### Install PMEM-CSI driver
+
+PMEM-CSI driver could be deployed to a Kubernetes cluster either using the
+PMEM-CSI operator or by using reference yaml files provided in source code.
+
+#### Install using the operator
+
+[PMEM-CSI operator](./design.md#operator-api) facilitates deploying and managing
+the [PMEM-CSI driver](https://github.com/intel/pmem-csi) on a Kubernetes cluster.
+
+- **Setup deployment certificates**
+
+Certificates are required as explained in [Security](./design.md#security).
```

Comment:
Switch the two paragraphs. Explain first that the operator does *not* need certificates because it will create them itself, then explain that this can be overridden with custom certificates.

Reply:
Rephrased the paragraph.